test/acceptance/features/steps/kind.py

```diff
@@ -115,7 +115,9 @@ def __deploy_primaza(self, kubeconfig_path: str, img: str):
         assert err == 0, f"error deploying Primaza's controller into cluster {self.cluster_name}"
 
     def __build_install_primaza_base_cmd(self, kubeconfig_path: str, img: str) -> Command:
+        # if HOME isn't set, intentionally fail
         return Command() \
+            .setenv("HOME", os.getenv("HOME")) \
```

Comment:
At least for mac this need USER adding too.

Reply:
Fixed; try it again and let me know if it works.

Reply:
Can confirm. Thanks.